package.json

```diff
@@ -152,7 +152,7 @@
         "react-spinners": "0.13.7",
         "react-use": "17.4.0",
         "remark-gfm": "3.0.1",
-        "resium": "1.15.1",
+        "resium": "^1.16.0",
```

Comment:
Please make sure you don't add `^` to the version so all devs are using the exact version. If we don't, we sometimes get a single dev with an issue with their local environment that is hard to troubleshoot together, among other reasons!

Reply:
I fixed in https://github.com/reearth/reearth-web/pull/375/commits/457ee177070917b78cd8f1af5b427d6a4cd78b55.
Thank you.